### PR TITLE
[CROW-38] Search campaign by title/subtitle

### PR DESCRIPTION
**Pull request by lucasgalarce**
# 📋 Board card

* [Backend: search project](https://loopstudio.atlassian.net/browse/CROW-38)

&nbsp;

# ℹ️ Description

Add filter to search campaign by title/subtitle

&nbsp;

# 🕵️ How was it tested?

- [x] Tested manually
- [x] Unit test passed and coverage threshold fulfilled

&nbsp;

### CODE REVIEW

**Review by marcosmartinez7 — APPROVED** [review]
lgtm !

File: src/features/campaigns/repositories/mongo/campaigns.repository.ts [hunk 1]
```diff
@@ -28,14 +28,20 @@ export class CampaignsMongoRepository {
...
     size: number;
     ownerId: string | null;
+    search: string | null;
   }) {
     const skipValue = page > 0 ? (page - 1) * size : 0;
     const filters = {
-      ...(ownerId && { owner: ownerId }),
+      owner: ownerId,
+      $or: [
+        { title: { $regex: `${search}`, $options: 'i' } },
```
Comment (renacargnelutti) [inline src/features/campaigns/repositories/mongo/campaigns.repository.ts:42]: NIT: Are the string literals necessaries?

**Review by renacargnelutti — APPROVED** [review]
💪 Just left a minor comment

File: src/features/campaigns/repositories/mongo/campaigns.repository.ts (hunk 1 above)
Comment (lucasgalarce) [reply src/features/campaigns/repositories/mongo/campaigns.repository.ts]: No. I delete it

**Pull request merged**